applications/solvers/additiveFoam/setDeltaT.H

```diff
@@ -4,20 +4,25 @@ if (adjustTimeStep)
     scalar maxAlphaCo =
         runTime.controlDict().lookupOrDefault<scalar>("maxAlphaCo", 0.1);
 
-    const scalar dtBydT = mag(runTime.deltaT() / (Tliq - Tsol)).value();
+    alphaCoNum = gMax(mag(T - T.oldTime())()) / (Tliq - Tsol).value();
 
-    alphaCoNum = gMax(mag(fvc::ddt(T))())*dtBydT;
+    Info << "alphaCoNum: " << alphaCoNum << endl;
 
-    Info << "Courant Number thermo: " << alphaCoNum << endl;
+    scalar maxFo = runTime.controlDict().lookupOrDefault<scalar>("maxFo", 1);
```

Comment:
In other OpenFOAM solid solvers, this is called `maxDi`. While it technically may be the Fourier number, should we adopt their syntax to be consistent with existing OF code?

Reply:
Added in [be374d3](https://github.com/ORNL/AdditiveFOAM/pull/16/commits/be374d33f2660d38271ac4de8dfdf082505c545e)